apps/docs/src/app/core/component-docs/off-screen/examples/off-screen-example.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'fd-off-screen-example',
+    template: `<div fdOffScreen>Current count value is: {{ count }}</div>
+        <br />
+        <button fd-button label="Increament count" (click)="onIncrease()"></button>
+        &nbsp;&nbsp;&nbsp;&nbsp;
+        <button fd-button label="Decreament count" (click)="onDecrease()"></button> `
+})
```

Comment:
This is good but can you add a more real case scenario? Maybe provide an example with the element that triggered the whole discussion around creating this directive. Thanks

Reply:
create combobix search example to show example.